testharness/src/main/java/com/google/accompanist/testharness/TestHarness.kt

```diff
@@ -70,6 +70,32 @@ import kotlin.math.floor
  * the [isScreenRound] will be left unchanged.
  */
 @Composable
+@Deprecated(
+    replaceWith = ReplaceWith(
```

Comment:
This isn't a perfect `ReplaceWith` - the resulting code will likely be far more verbose than necessary, with the assumption that usually only one or a couple parameters were specified, but it brings in the imports and can then be edited down to the important ones.